src/main/scala/pirate/Read.scala

```diff
@@ -180,7 +180,14 @@ object Read extends shapeless.ProductTypeClassCompanion[Read] {
     def append(p1: Read[A], p2: => Read[A]): Read[A] = p1 ||| p2
   }
 
-  implicit def ReadTypeClass: shapeless.ProductTypeClass[Read] = new shapeless.ProductTypeClass[Read] {
+  /*
+   * WARNING!!! Don't change the name `typeClass`.
+   * It has to be `typeClass` to be compatible with Shapeless 2.1 and higher.
+   *
+   * Reference:
+   * https://github.com/milessabin/shapeless/blob/shapeless-2.1.0/core/src/main/scala/shapeless/typeclass.scala#L48
+   */
+  implicit val typeClass: shapeless.ProductTypeClass[Read] = new shapeless.ProductTypeClass[Read] {
```

Comment:
Not sure you need such a large warning. Just add `override` and remove the comment?

Reply:
It's not `override` unless Shapeless is upgraded to `2.1.0` or higher. Should I upgrade Shapeless from `2.0.0` to `2.1.0`?

Reply:
As discussed, I'll upgrade it to `2.1.0`.